WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -408,7 +410,10 @@ class AztecPostViewController: UIViewController, PostEditor {
     fileprivate lazy var verificationPromptHelper: AztecVerificationPromptHelper? = {
         return AztecVerificationPromptHelper(account: self.post.blog.account)
     }()
-
+    
+    // Autosaving posts 
+    fileprivate var shouldAutoSave: Bool = false
+    fileprivate lazy var autoSaveDebouncer = Debouncer(delay: 0.5)
```

Comment:
We like to bundle up values like the `0.5` here in a struct (or structs) at the bottom of the file. You'll find `struct Constants` around like `3246`. That's probably a good place to put this 😀